capsules/extra/src/ieee802154/mac.rs

```diff
@@ -172,20 +171,21 @@ impl<'a, R: radio::Radio<'a>> radio::RxClient for AwakeMac<'a, R> {
         if let Some((_, (header, _))) = Header::decode(&buf[radio::PSDU_OFFSET..], false).done() {
             if let Some(dst_addr) = header.dst_addr {
                 addr_match = match dst_addr {
-                    MacAddress::Short(addr) => addr == self.radio.get_address(),
+                    MacAddress::Short(addr) => {
+                        // Check if address matches radio or is set to multicast short addr 0xFFFF
+                        (addr == self.radio.get_address()) || (addr == 0xFFFF)
+                    }
                     MacAddress::Long(long_addr) => long_addr == self.radio.get_address_long(),
                 };
             }
         }
-
         if addr_match {
-            //debug!("[AwakeMAC] Rcvd a 15.4 frame addressed to this device");
+            // debug!("[AwakeMAC] Rcvd a 15.4 frame addressed to this device");
```

Comment:
Maybe this is a good compromise: leave the debug statements there, but commented out. They're essentially comments, but you can activate them again if you are debugging.

Reply:
I think I actually like the code as-is — the common-case / correct-case debug statements are commented out, so in normal operation the console isn't flooded. Elsewhere, when errors / uncommon cases occur, debugs are still printed by default.